.github/workflows/helm-check.yaml

```diff
@@ -42,14 +41,40 @@ jobs:
           cd install/kubernetes/cilium
           helm lint --with-subcharts --values values.yaml
 
+      - name: Build docker images
+        run: |
+          make docker-image-no-clean
+          make docker-operator-image
+
+      - name: Prevent K8s from pulling images
+        run: |
+          sed -i 's;pullPolicy: Always;pullPolicy: Never;g' install/kubernetes/cilium/values.yaml
```

Comment:
Can't we set the pull policy value with helm?

Reply:
Not with the action we are using here :/